src/main/resources/assets/tconstruct/book/materials_and_you/sections/modifiers.json

```diff
@@ -1,10 +1,128 @@
 [
+  {
+	"name": "general",
+    "type": "text",
+    "data": "modifiers/general.json"
+  },
+   {
+    "type": "modifier",
+    "data": "modifiers/diamond.json"
+  },
+    {
+    "type": "modifier",
+    "data": "modifiers/emerald.json"
+  },
+    {
+    "type": "modifier",
+    "data": "modifiers/experienced.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/netherite.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/reinforced.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/soulbound.json"
+  },
+  {
+    "name": "harvest",
+    "type": "text",
+    "data": "modifiers/harvest.json"
+  },
+    {
+    "type": "modifier",
+    "data": "modifiers/blasting.json"
+  },
   {
     "type": "modifier",
     "data": "modifiers/haste.json"
   },
   {
     "type": "modifier",
-    "data": "modifiers/beheading.json"
+    "data": "modifiers/hydraulic.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/lightspeed.json"
+  },
+  {
+    "name": "weapon",
+    "type": "text",
+    "data": "modifiers/weapon.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/antiaquatic.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/baneofarthropods.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/cooling.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/fiery.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/knockback.json"
+  },
+  {
+    "type": "modifier",
+    "data": "modifiers/magnetic.json"
```

Comment:
Magnetic is a lot more useful for harvest than weapon, so feels wrong in the weapon category. I'd suggest general probably.